Gruntfile.js

```diff
@@ -77,20 +77,14 @@ grunt.initConfig({
         spawn: false
       }
     }
-  },
-  templateSize: {
-    benchmark: {
-      src: ['benchmark/templates/*.td']
-    }
   }
 });
 
-grunt.task.loadTasks('grunt-tasks/');
 grunt.loadNpmTasks('grunt-mocha');
 
 grunt.registerTask('dist', ['peg', 'eslint', 'babel:dist']);
 grunt.registerTask('test', ['dist', 'babel:acceptance', 'browserify:test', 'mocha']);
 grunt.registerTask('sandbox', ['dist', 'browserify:sandbox']);
-grunt.registerTask('benchmark', ['dist', 'bytesize', 'templateSize']);
+grunt.registerTask('benchmark', ['dist', 'bytesize']);
```

Comment:
We might want it at some point, but it's not necessary now. We can delete now and bring it back from git history when/if it's needed.